packages/shared/src/components/utilities.tsx

```diff
@@ -249,3 +254,9 @@ export const getContextBottomPosition = (
 
   return { x: right, y: bottom + 4 };
 };
+
+export const fadedBackgroundStyle: CSSProperties = {
```

Comment:
Can you move this to a css class instead?

Reply:
Good point. Let me find a global CSS file we use.

Reply:
Is this still open?

Reply:
Yep. My bad. I must have stashed it somewhere. Should be fixed now. Thank you for the notice! 🚢